### PR TITLE
Adding test connection  to clickhouse destination

### PR DESCRIPTION
**Pull request by Luishfs**
# Summary
Adding test connection to clickhouse destination
Added better process function
updated logging variables

# Tests
Tested using mage UI

Success case:
Screenshot from 2023-08-02 07-56-47

Config error:
Screenshot from 2023-08-02 07-57-26

Connection Error ( database not found):
Screenshot from 2023-08-02 07-57-54

cc:
@wangxiaoyou1993

### CODE REVIEW

File: mage_integrations/mage_integrations/destinations/clickhouse/target_clickhouse/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Clickhouse
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/clickhouse/target_clickhouse/README.md:1]: Can you put the README file under this folder? https://github.com/mage-ai/mage-ai/tree/master/mage_integrations/mage_integrations/destinations/clickhouse

File: mage_integrations/mage_integrations/destinations/clickhouse/__init__.py [hunk 2]
```diff
@@ -1,22 +1,42 @@
...
+from clickhouse_sqlalchemy import make_session
 from mage_integrations.destinations.base import Destination
 from mage_integrations.destinations.clickhouse.target_clickhouse.target import (
     TargetClickhouse,
 )
 
 
 class Clickhouse(Destination):
-    def process(self, input_buffer) -> None:
+    def process(self) -> None:
```
Comment (wangxiaoyou1993) [inline mage_integrations/mage_integrations/destinations/clickhouse/__init__.py:12]: Let's not change the method signature. https://github.com/mage-ai/mage-ai/blob/master/mage_integrations/mage_integrations/destinations/base.py#L295

File: mage_integrations/mage_integrations/destinations/clickhouse/target_clickhouse/README.md (hunk 1 above)
Comment (Luishfs) [reply mage_integrations/mage_integrations/destinations/clickhouse/target_clickhouse/README.md]: totally missed that, sorry. Moved in 947b0a65dc6207834bed85c022bd8d8efd608b13

File: mage_integrations/mage_integrations/destinations/clickhouse/__init__.py (hunk 2 above)
Comment (Luishfs) [reply mage_integrations/mage_integrations/destinations/clickhouse/__init__.py]: Modified on 947b0a65dc6207834bed85c022bd8d8efd608b13

**Review by wangxiaoyou1993 — APPROVED** [review]

**Pull request merged**